Gemfile

```diff
@@ -4,8 +4,8 @@ source 'https://rubygems.org'
 gem 'berkshelf', '~> 3.0'
 
 # Install omnibus software
-gem 'omnibus', '5.5'
-gem 'omnibus-software', :github => 'opscode/omnibus-software' #, :branch => 'omnibus/3.2-stable'
+gem 'omnibus', '5.4'
+gem 'omnibus-software', :github => 'qingling128/omnibus-software', :branch => 'omnibus-5.4.0'
```

Comment:
I forked this branch just to unblock myself from testing. If we do decide to do it this way, I will create a formal branch.